utilities/pipelines/resourcePublish/Get-ModulesToUpdate.ps1

```diff
@@ -0,0 +1,373 @@
+﻿#region Helper functions
+
+<#
+.SYNOPSIS
+Get modified files between two commits.
+
+.PARAMETER Commit
+A git reference to base the comparison on.
+
+.PARAMETER CompareCommit
+A git reference to compare with.
+
+.EXAMPLE
+Get-ModifiedFiles -Commit "HEAD^" -CompareCommit "HEAD"
+
+    Directory: C:\Repo\Azure\ResourceModules\utilities\pipelines\resourcePublish
+
+Mode                 LastWriteTime         Length Name
+----                 -------------         ------ ----
+la---          08.12.2021    15:50           7133 Script.ps1
+
+Get modified files between previous and current commit.
+#>
+function Get-ModifiedFiles {
```

Comment:
Rename to singular as per PowerShell best practices:

```suggestion
function Get-ModifiedFileList {
```

Note: Change all references

Reply:
I didn't apply this commit as there were multiple other places that needed updating. Fixed.